src/Core/Commander/ManagerCommands.js

```diff
@@ -52,6 +59,7 @@ define('Core/Commander/ManagerCommands', [
 
         ManagerCommands.prototype.addCommand = function(command) {
             this.queueAsync.queue(command);
```

Comment:
I'd suggest doing this instead:
```js
if (commandIsRunnable(command)) {
  executeCommand(command);
} else {
  this.queueAsync.queue(command);
}
```

(no need to queue/dequeue an immediately runnable command)